scripts/client/_otter_env_core.sh

```diff
@@ -134,12 +134,11 @@ case "$osname" in
     Darwin)
         anasoft_distro=
         otter_perl=
-        PERL5LIB="${PERL5LIB}:\
-$anasoft/lib/site_perl:\
-$anasoft/lib/perl5/site_perl:\
-$anasoft/lib/perl5/vendor_perl:\
-$anasoft/lib/perl5\
-"
+        #This is based on someone having a Mac will have the app installed. Hopefully it should work even
+        #if it is not installed.
```

Comment:
I am not entirely sure what this is supposed to mean, could you possibly rephrase?

Reply:
Not really sure how I can rephrase that. The changes for the code are made so a DMG file can be created and the annotators can simply copy the file anywhere and have the client ready. The DMG file will be created from user directories, the paths in INC will start with '/Users', which will ntot be found in the annotators computer. If the client is not coming from a DMG file, hopefully the INC directories will be the correct ones and the substitution would do nothing

Reply:
This really reads wrong to me, though. Maybe it's just the matter of grammar.